test/Npgsql.Tests/Types/InternalTypeTests.cs

```diff
@@ -96,6 +96,37 @@ public async Task PgLsn()
 
         #endregion NpgsqlLogSequenceNumber / PgLsn
 
+        #region NpgsqlTransactionId / PgTxn8
+
+        static readonly TestCaseData[] EqualsTransactionIdCases = {
+            new TestCaseData(new NpgsqlTransactionId(1ul), null).Returns(false),
+            new TestCaseData(new NpgsqlTransactionId(1ul), new object()).Returns(false),
+            new TestCaseData(new NpgsqlTransactionId(1ul), 1ul).Returns(false), // no implicit cast
+            new TestCaseData(new NpgsqlTransactionId(1ul), new NpgsqlTransactionId(1ul)).Returns(true),
+        };
+
+        [Test, TestCaseSource(nameof(EqualsTransactionIdCases))]
+        public bool NpgsqlTransactionIdEquals(NpgsqlTransactionId lsn, object? obj)
```

Comment:
The NpgsqlTransactionId seems to be missing from this PR, did you forget to add it to the commit?

Reply:
Yeah, sorry! I should have added...